src/httpClient/index.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const APPLICATION_JSON = 'application/json';
+const CONTENT_TYPE = 'Content-Type';
+
+const client = axios.create({
+  baseURL: 'http://localhost:3000/',
```

Comment:
WDYT of setting the deployed BE right away?

 ```suggestion
  baseURL: 'https://assistive-matcher-poc.herokuapp.com/',
```